atexit.go

```diff
@@ -35,17 +35,28 @@ import (
 
 const (
 	// Version is package version
-	Version = "0.2.0"
+	Version = "0.3.0"
 )
 
-var handlers = []func(){}
+type HandlerId uint
+
+func (h *HandlerId) Get() HandlerId {
+	defer func() {
+		*h++
+	}()
+
+	return *h
+}
+
+var handlers = map[HandlerId]func(){}
 var handlersLock sync.RWMutex
 var once sync.Once
+var nextFuncId HandlerId
 
 func runHandler(handler func()) {
 	defer func() {
 		if err := recover(); err != nil {
-			fmt.Fprintln(os.Stderr, "error: atexit handler error:", err)
+			_, _ = fmt.Fprintln(os.Stderr, "error: atexit handler error:", err)
```

Comment:
What's the value of `_, _ =`?

Reply:
fmt.Fprintln return `n` and `err` that's why i use `_, _ =` to say explicitly that we want to ignore returned values